Lib/LinkedIn/LinkedInApi.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+
+use GuzzleHttp\Client;
+
+class LinkedInApi {
```

Comment:
Seein that the repository is 3 years old and the last update is 9 months ago, it looks pretty active to me.

However, I disagree that this makes things more complex. In fact, the contrary is true. Creating and maintaining an API library ourselves is far more complex than using one.